x-pack/packetbeat/npcap/installer/README

```diff
@@ -0,0 +1,7 @@
+This directory is pinned to allow placement of the Npcap OEM installer
+during x-pack Packetbeat builds.
+
+Only one exe file may be placed here and this should be the most recent
+available Npcap installer. As part of the distribution a place-holder file
+is put here to allow compilation. This file must be removed when a real
+installer in included for the build.
```

Comment:
Let's add a `.gitignore` to this directory so that no real exe files are accidentally committed.

```
# Ignore everything but the place-holder.
*
!npcap-0.00.exe
```

Reply:
Committing this file required a `-f` and will now be tracked without opening the flood gates for other .exe files, but I'll add this for human consumption.